docs/installation.rst

```diff
@@ -1,36 +1,37 @@
 Installation
 ========================
 
-This repository contains a folder named `Rubrik`_. The folder needs to be installed into one of your PowerShell Module Paths using one of the installation methods outlined in the next section. Common PowerShell module paths include:
+This repository contains a folder named `Rubrik`_. The folder needs to be installed into one of your PowerShell Module Paths using one of the installation methods outlined in the next section. To see the full list of available PowerShell Module paths, use ``$env:PSModulePath.split(';')`` in a PowerShell console.
 
-1. Current User: ``%USERPROFILE%\Documents\WindowsPowerShell\``
-2. All Users: ``%WINDIR%\System32\WindowsPowerShell\v1.0\``
+Common PowerShell module paths include:
 
-Option 1: Installer Script
+1. Current User: ``%USERPROFILE%\Documents\WindowsPowerShell\Modules\``
+2. All Users: ``%WINDIR%\System32\WindowsPowerShell\v1.0\Modules\``
```

Comment:
Shouldn't it be %ProgramFiles%\WindowsPowerShell\Modules? The System32 path should only be used for stuff from Microsoft.